Documentation/kubelet-wrapper.md

```diff
@@ -0,0 +1,39 @@
+# Kubelet Wrapper Script
+
+The kubelet has some unique requirements, so we need to be able to run the kubelet in an unconstrained environment. However, we also want to ship the kubelet as a container image to take advantage of all that has to offer (image discovery, signing/verification, management).
```

Comment:
Maybe be a little more straightforward with this?

> The kubelet is the orchestrator of containers on each host in the Kubernetes cluster &mdash; it starts and stops containers, manipulates iptables rules, and other low-level, essential tasks. In order to accomplish these tasks, the kubelet requires special permissions on the host.

> CoreOS recommends running the kubelet using the rkt container engine, because it has the correct set of features to enable these special permissions, while taking advantage of all that container packaging has to offer: image discovery, signing/verification, and simplified management.